contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -2513,13 +2521,49 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 							 * Postgres does not allow us to create functions with different return types
 							 * so we need to delete and recreate them 
 							 */
-							RemoveFunctionById(oldoid);
+							performDeletion(&originalFunc, DROP_RESTRICT, 0);
```

Comment:
What's the difference between those two function ?

Reply:
`doDeletion` performs a full deletion of a single object. `RemoveFunctionById` is only part of the deletion and does not cleanup all of the objects created by a multiline tvf function. `RemoveFunctionById` was sufficient for procs, basic funcs, and inline tvf funcs, but not multiline because they create table relations that do not get deleted by `RemoveFunctionById`